src/mito2/src/manifest/action.rs

```diff
@@ -120,6 +120,7 @@ pub struct RegionManifest {
     pub flushed_entry_id: EntryId,
     /// Last sequence of flushed data.
     pub flushed_sequence: SequenceNumber,
+    pub committed_sequence: Option<SequenceNumber>,
```

Comment:
Maybe need to add `#[serde(default)]` here. Do we have a JSON compatibility test for this struct?